packages/lesswrong/components/walledGarden/PortalBarGcalEventItem.tsx

```diff
@@ -17,30 +17,34 @@ const styles = (theme) => ({
     width: 120,
     textAlign: "right",
     display: "inline-block"
-  },
-  tooltip: {
-    whiteSpace: "pre-wrap"
   }
 })
 
-const UrlClass = getUrlClass()
 
-const PortalBarGcalEventItem = ({classes, gcalEvent}) => {
+export const getAddToCalendarLink = (gcalEvent) => {
   const { LWTooltip } = Components
-
+  
+  const UrlClass = getUrlClass()
   const url = new UrlClass(gcalEvent.htmlLink)
   const eid = url.searchParams.get("eid")
   const addToCalendarLink = `https://calendar.google.com/event?action=TEMPLATE&tmeid=${eid}&tmsrc=${gcalEvent.organizer.email}`
   const link = <a href={addToCalendarLink} target="_blank" rel="noopener noreferrer">
     {gcalEvent.summary}
   </a>
 
+  if (gcalEvent.description) {
+    return <LWTooltip title={<div dangerouslySetInnerHTML={{__html:gcalEvent.description}}/>}>
```

Comment:
Needs sanitization

Reply:
(This one would've been actual XSS, I think. The name `dangerouslySetInnerHTML` is there to warn you that it should only be used on something that you're pretty certain has been sanitized.)